src/middleware/warp.ts

```diff
@@ -57,8 +57,17 @@ export const warp = WarpFactory.forMainnet(
         ...defaultCacheOptions,
         dbLocation: `./cache/warp/lmdb/kv/${contractTxId}`,
       }),
+  )
+  .useContractCache(
+    new SqliteContractCache({
+      ...defaultCacheOptions,
+      dbLocation: `./cache/warp/sqlite/contracts`,
+    }),
+    new LmdbCache({
```

Comment:
SQLite implementation doesn't satisfy this interface, so using LMDB > LevelDb for performance